src/pages/billing/Project.js

```diff
@@ -81,11 +82,11 @@ const ExpandedInfoRow = ({ title, details, errorMessage }) => {
   ])
 }
 
-const WorkspaceCard = memoWithName('WorkspaceCard', ({ workspace, billingAccountStatus, isExpanded, onExpand }) => {
+const WorkspaceCard = memoWithName('WorkspaceCard', ({ workspace, billingProject, billingAccountStatus, isExpanded, onExpand }) => {
   const { namespace, name, createdBy, lastModified, googleProject, billingAccountDisplayName, billingAccountErrorMessage } = workspace
   const workspaceCardStyles = {
     field: {
-      ...Style.noWrapEllipsis, flex: 1, height: '1rem', width: `calc(50% - ${(workspaceLastModifiedWidth + workspaceExpandIconSize) / 2}px)`, paddingRight: '1rem'
+      ...Style.noWrapEllipsis, flex: 1, height: '1.20rem', width: `calc(50% - ${(workspaceLastModifiedWidth + workspaceExpandIconSize) / 2}px)`, paddingRight: '1rem'
```

Comment:
This is to prevent the focus indicator from getting truncated.

Before:
![image](https://user-images.githubusercontent.com/484484/189415374-5d6311e5-0714-4e9c-9089-138e7235deec.png)